docs/Deployment.md

```diff
@@ -46,9 +48,11 @@ juju config serial-vault api_keys=Heib2vah2aen3ai
 # Configure the admin service
 #   keystore_secret: part of the key used that is used to encrypt the stored data
 #   api_keys: the key that must be provided in the header of the web service requests
+#	csrf_auth_key: the key to protect server routing from csrf attacks
```

Comment:
I've set "32 bytes long key to protect server from cross site request forgery attacks" as text. Is that the one you mean?